src/taipy/gui_core/GuiCoreLib.py

```diff
@@ -54,7 +54,7 @@ def __repr__(self):
 DataNode.__bases__ += (_GCDoNotUpdate,)
 Job.__bases__ += (_GCDoNotUpdate,)
 
-Config.configure_global_app(read_entity_retry=3)
+Config.configure_core(read_entity_retry=3)
```

Comment:
I believe this should be in taipy-core the default CoreSection value.
GuiCoreLib should not have to care about taipy core configuration. What do you think?

Reply:
The current default value of `read_entity_retry` for now is None 🤣

Since this is a special case only use in GuiCoreLib, I think it can be here.
But if the user always run the GuiCoreLib, I'm fine with moving it to Core.

Reply:
I would set the default value to 3 in taipy-core and remove this. It should have no impact on the performance. Plus that could cause the problem even if we don't use CoreGui visual elements.

Reply:
Sure. I'm on it

Reply:
This line has been removed. The responsibility has been moved to taipy-core on PR https://github.com/Avaiga/taipy-core/pull/689.